components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/service/DCRManagementService.java

```diff
@@ -225,16 +219,6 @@ private RegistrationResponseProfile createOAuthApplication(RegistrationRequestPr
                 log.debug("Created OAuth App " + applicationName);
             }
 
-            try {
-                if (isHashDisabled) {
-                    createdApp = oAuthAdminService
-                            .getOAuthApplicationDataByAppName(oAuthConsumerApp.getApplicationName());
-                }
-            } catch (IdentityOAuthAdminException e) {
-                throw IdentityException.error(DCRException.class, ErrorCodes.BAD_REQUEST.toString(), e.getMessage());
-
-            }
-
             if (log.isDebugEnabled()) {
                 log.debug("Retrieved Details for OAuth App " + createdApp.getApplicationName());
```

Comment:
Shall we integrate this debug log with the previous debug log.isDebugEnabled in line[1]?

[1] https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/886/files#diff-7990f976553c35088679eee6058b2616R218